### PR TITLE
Update 2024-08-14-racket-v8-14.md

### PR DESCRIPTION
**Pull request by spdegabrielle**
Don’t pull yet I need to do more

### CODE REVIEW

File: blog/_src/posts/2024-08-14-racket-v8-14.md [hunk 1]
```diff
@@ -8,17 +8,21 @@
...
   program rather than an external tool. Using the language approach
   cooperates with other Racket tools and allows demodularization to be
   used on libraries. Additionally, demodularization handles submodules
   and can prune definitions, significantly shrinking the size of some
-  programs.
+  programs.  ['#lang compiler/demod'](https://docs.racket-lang.org/raco/demod.html#%28mod-path._compiler%2Fdemod%29)
 
 - The `contract-in` form enables contracts to be specified in an
-  importing module.
+  importing module. [8.6 Attaching Contracts to Values]([)](https://docs.racket-lang.org/reference/attaching-contracts-to-values.html#(part._attaching-contracts-to-values))
```
Comment (sorawee) [inline blog/_src/posts/2024-08-14-racket-v8-14.md:25]: This doesn't look like a valid syntax to create Markdown link to me...

Also, should the whole link be inside parentheses? It looks weird to suddenly have "8.6 Attaching Contracts to Values" out of nowhere.

**Comment by spdegabrielle** [comment]
@jbclements good to go

File: blog/_src/posts/2024-08-14-racket-v8-14.md [hunk 2]
```diff
@@ -1,45 +1,49 @@
...
+  importing module. ([8.6 Attaching Contracts to Values](https://docs.racket-lang.org/reference/attaching-contracts-to-values.html#%28form._%28%28lib._racket%2Fcontract%2Fbase..rkt%29._contract-in%29%29))
 
 - The `#%declare` form supports the `#:flatten-requires` and
-  `#:unlimited-compile` keywords
+  `#:unlimited-compile` keywords ([`#%declare`](https://docs.racket-lang.org/reference/module.html#%28form._%28%28quote._~23~25kernel%29._~23~25declare%29%29))
 
 - Identifiers such as `case-λ`, `match-λ` and `λ` are equivalent to
   their Latin-alphabet equivalents (`case-lambda`, `match-lambda`, and
-  `lambda`) in more contexts, including uses of `match`.
+  `lambda`) in more contexts, including uses of `match`. ([]())
```
Comment (sorawee) [inline blog/_src/posts/2024-08-14-racket-v8-14.md:32]: Do you intend to add a link here? If not, let's remove it.

File: blog/_src/posts/2024-08-14-racket-v8-14.md [hunk 3]
```diff
@@ -1,45 +1,49 @@
...
 - The `hash-filter` function allows the use of a predicate to select a
-  sub-table of a hash table.
+  sub-table of a hash table.([`hash-filter`](https://docs.racket-lang.org/reference/hashtables.html#%28def._%28%28lib._racket%2Fhash..rkt%29._hash-filter%29%29))
 
 - The module browser can perform filtering on submodules.
 
 - The `raco test` command-line form respects the
   `current-test-invocation-directory` parameter, and shows well-formed
-  module paths in its output.
+  module paths in its output.((`current-test-invocation-directory`)[https://docs.racket-lang.org/raco/test.html#%28def._%28%28lib._raco%2Ftesting..rkt%29._current-test-invocation-directory%29%29])
```
Comment (sorawee) [inline blog/_src/posts/2024-08-14-racket-v8-14.md:41]: For links, square brackets should appear first, and then parens.

File: blog/_src/posts/2024-08-14-racket-v8-14.md [hunk 4]
```diff
@@ -1,45 +1,49 @@
...
+
 - 64-bit ARM Windows is one of the standard pre-built downloads.
 
 - The `compiler/demod` language integrates demodularization into the
   program rather than an external tool. Using the language approach
   cooperates with other Racket tools and allows demodularization to be
   used on libraries. Additionally, demodularization handles submodules
   and can prune definitions, significantly shrinking the size of some
-  programs.
+  programs.  [`#lang compiler/demod`](https://docs.racket-lang.org/raco/demod.html#%28mod-path._compiler%2Fdemod%29)
```
Comment (sorawee) [inline blog/_src/posts/2024-08-14-racket-v8-14.md:22]: Do we want to add parens around this link too?

Comment (jbclements) [reply blog/_src/posts/2024-08-14-racket-v8-14.md:22]: I'm going to try rendering, and then make any needed fixes. Thank you so much for your work on this.

**Comment by jbclements** [comment]
Probably should have squashed first, mea culpa.

**Pull request merged**